packages/firestore/test/util/api_helpers.ts

```diff
@@ -42,6 +42,7 @@ import {
   key,
   path as pathFrom
 } from './helpers';
+import { Provider, ComponentContainer } from '@firebase/component';
```

Comment:
Can you verify this is expected?  Or is it a bad merge or something?

Reply:
The only thing that was imported was the `defaultDocumentDataConverter`, which prompted the linter to split these into separate lines.